src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -225,15 +225,24 @@ namespace winrt::TerminalApp::implementation
     void CommandPalette::_selectedCommandChanged(const IInspectable& /*sender*/,
                                                  const Windows::UI::Xaml::RoutedEventArgs& /*args*/)
     {
+        const auto currentlyVisible{ Visibility() == Visibility::Visible };
```

Comment:
ho boy I think this change might predate me doing this as a separate control. 1449088e80b81d82cbf5456baaf69798ee849d16, d3b5533a1eef45edbfbae4e68ee7e9507c97406c and 0bda66fc2f0f5d6a5e709d8ebc4678256d132fc3 are not super descriptive 😓

Reply:
I'm gonna back this out for now, since I don't _think_ we need it. 

I believe this was to manually fire a `PreviewAction` to end the preview... but I dunno when that was actually relevant.